### PR TITLE
Create pay-equity-triangle-nc.md

### PR DESCRIPTION
**Pull request by realjenny**

### CODE REVIEW

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Google’s decision to cut salaries for workers follows the centuries old trend of corporations turning to the South and to workers of color as a way to cut costs by lowering
+worker pay. Google currently employs 300 workers in the region and plans to expand to 1,000 employees in the near future. The current workforce is a mix of previous hires
+local to the area, transfers to the region, and recent hires. 
+
+“Once again, Google fails to commit to their declarations of equity and inclusion when it comes to their bottom line. Google has an opportunity to hire a diverse workforce
+and compensate these workers in step with the national pay band for other Google workers. Instead, Google has decided to cut the salaries of all workers already in the region,
+in order to set a lower pay standard for all incoming hires. This is unacceptable. All workers are contributing to Alphabet’s astronomic growth and thus should receive their
+fair share and be fully involved in all decisions regarding worker compensation,” said Ashok Chandwaney, Google software engineer and member of the executive committee of the
+Alphabet Workers Union-CWA.
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:49]: AWU isn’t acronym’d here

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: 'AWU-CWA Workers Demand Pay Equity as Google Cuts Worker Pay in Offices Aimed at Diversity Recruitment'
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:2]: Can I suggest a reword for flow?

“AWU-CWA Workers Demand Pay Equity for Offices Aimed at Diversity Recruitment, as Google Cuts Worker Pay”
or more controversially...
“AWU-CWA Workers Demand Pay Equity for Diversity Recruitment Offices, as Google Cuts Worker Pay”

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+_February 18, 2022 at 7am ET_
+
+_Press contact: press@alphabetworkersunion.org_
+
+# AWU-CWA Workers Demand Pay Equity as Google Cuts Worker Pay in Offices Aimed at Diversity Recruitment
+
+In the Triangle area of North Carolina, Google workers of the Alphabet Workers Union-CWA launched the [Triangle Comp Grievance Letter](https://airtable.com/shrWrnlxqumeJhF1g)
+internally to Google on Monday, February 14, 2022. Workers from the Durham and Chapel Hill Google offices in North Carolina came together to demand pay equity for current and
+future employees in the area. Over a quarter of employees across the local office sites have signed the grievance letter along with Google workers across the country,
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:20]: Google workers or Alphabet workers?

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+resulting in hundreds of signatures. 
+
+The Triangle area refers to the metropolitan area of North Carolina that is anchored by Raleigh, Durham, and Chapel Hill. Workers published the letter in response to
+Google announcing that Google workers in the Triangle area would be moved down to the “discount” pay band, the lowest pay band in the nation. This pay band is lower than the
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:24]: s/Google workers/workers

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+“national” pay band and several steps below the “premium” pay band for New York and the Bay Area. The band for the Triangle area is lower than workers in comparable locations
+like Kenosha, Wisconsin;  Atlanta, Georgia; Miami, Florida; and even lower than remote workers in rural counties, nearby or in Virginia. Some workers will experience as much
+as a 25% pay cut. Google has refused to be transparent or engage with workers regarding this decision. 
+
+Google extensively publicized the development of the Google Durham office as a concerted effort to diversify the workforce through the targeted recruitment of local HBCU
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:29]: s/Google extensively/Google has extensively

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+grads and other historically excluded populations from the Google workforce. The announcement of the salary cuts calls into question Google’s commitment to ensuring current
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:30]: if you wanna be snippy u could make it “other populations historically excluded from the Google workforce”

File: frontend/content/press/releases/pay-equity-triangle-nc.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+Workers are calling on management to reverse the decision to cut pay for workers in the region and commit to equitable compensation for current and future workers. 
+
+“We have worked hard, in spite of the difficulties of these last two years, and Alphabet has enjoyed record profits thanks to our work. In response, Alphabet
+has decided to cut salaries for workers in the Triangle region. **Why does Google think it is acceptable to choose our workplace—which is intended to be a
+hub for diversity—to be the site where workers are receiving the lowest salary ranges in the entire country?** If Google truly means for the Triangle region to be
+a hub for diversity and local economic development, they should reverse the current pay cut and commit to full transparency with workers regarding any future compensation
+decisions,” said a worker at the Durham Google office site.
+
+Google’s decision to cut salaries for workers follows the centuries old trend of corporations turning to the South and to workers of color as a way to cut costs by lowering
```
Comment (auni53) [inline frontend/content/press/releases/pay-equity-triangle-nc.md:41]: ** centuries-old, I think

**Review by butterflyhug — APPROVED** [review]

**Pull request merged**